Dockerfile

```diff
@@ -1,25 +1,17 @@
-# gradle:7.0.0-jdk16
-FROM gradle@sha256:d31e12d105e332ec2ef1f31c20eac6d1467295487ac70e534e3c1d0ae4a0506e as builder
+FROM gradle:7.3.3-jdk17 as builder
```

Comment:
Didn't use the sha since it was not used for the image below, and I fought a bit with intel/arm architecture.